test/extensions/zygote.jl

```diff
@@ -0,0 +1,19 @@
+using GPUArraysCore: GPUArraysCore
```

Comment:
```suggestion
using GPUArrays
```

GPUArrays re-exports the Core functionality.